Bhagavathi_Chirra/week1/ifelse-exercises/js/exercise1.js

```diff
@@ -0,0 +1,127 @@
+// What numbers bigger?
+// Write a function named greaterNum that:
+//
+// takes 2 arguments, both numbers.
+// returns whichever number is the greater (higher) number.
+// Call that function 2 times with different number pairs, and log the output to make sure it works (e.g. "The greater number of 5 and 10 is 10.").
+
+const greaterNum = function(number1,number2){
+  if(number1 > number2){
+    return `${number1} is big`;
+  }
+  else
```

Comment:
Please get into the habit of wrapping the else statement inside of a block - 
+  if( number1 > number2 ){
+    return `${number1} is big`;
+  } else {
    }